corehq/form_processor/backends/sql/update_strategy.py

```diff
@@ -222,6 +226,11 @@ def _apply_attachments_action(self, attachment_action, xform):
                     self.case.track_delete(current_attachments[name])
             elif att.attachment_src:
                 form_attachment = xform.get_attachment_meta(att.attachment_src)
+                if form_attachment is None:
+                    # Probably an improperly configured form. We need a way to
+                    # convey errors like this to domain admins.
```

Comment:
Nit: Not sure this is a very useful place for this comment, sounds more like a backlog item if you think someone actually needs to do this.

Reply:
Fair point. In addition to being a feature suggestion, this comment is also informative that errors occurring here may be user error and possibly not directly actionable by HQ devs.